index.html

```diff
@@ -17,8 +21,8 @@ <h1>Happy Hearts Roaming</h1>
           </a>
         </div>
         <div class="navbar">
-          <a href="myStory.html">MY STORY</a>
-          <a href="#blogs">BLOGS</a>
+          <div class="mystory-header"><a href="myStory.html">MY STORY</a></div>
```

Comment:
The nav items should have height equal to the header

Reply:
I adjusted and created another PR. Please have a look.